src/js/eva-annotation-config.js

```diff
@@ -0,0 +1,1004 @@
+    },
+    'upstream_gene_variant': {
+        name: 'upstream_gene_variant',
+        acc:'SO:0001631',
+        color: '#A2B5CD',
+        impact: 'MODIFIER',
+        description:'A sequence variant located 5\' of a gene'
+    },
+    'downstream_gene_variant': {
+        name: 'downstream_gene_variant',
+        acc:'SO:0001632',
+        color: '#A2B5CD',
+        impact: 'MODIFIER',
+        description:'A sequence variant located 3\' of a gene'
+    },
+    'TFBS_ablation': {
+        name: 'TFBS_ablation',
+        acc:'SO:0001895',
+        color: '#A52A2A',
+        impact: 'MODERATE',
+        description:'A feature ablation whereby the deleted region includes a transcription factor binding site'
+    },
+    'TFBS_amplification': {
+        name: 'TFBS_amplification',
+        acc:'SO:0001892',
+        color: '#A52A2A',
+        impact: 'MODIFIER',
+        description:'A feature amplification of a region containing a transcription factor binding site'
+    },
+    'TF_binding_site_variant': {
+        name: 'TF_binding_site_variant',
+        acc:'SO:0001782',
+        color: '#A52A2A',
+        impact: 'MODIFIER',
+        description:'A sequence variant located within a transcription factor binding site'
+    },
+    'regulatory_region_ablation': {
+        name: 'regulatory_region_ablation',
+        acc:'SO:0001894',
+        color: '#A52A2A',
+        impact: 'MODERATE',
+        description:'A feature ablation whereby the deleted region includes a regulatory region'
+    },
+    'regulatory_region_amplification': {
+        name: 'regulatory_region_amplification',
+        acc:'SO:0001891',
+        color: '#A52A2A',
+        impact: 'MODIFIER',
+        description:'A feature amplification of a region containing a regulatory region'
+    },
+    'regulatory_region_variant': {
+        name: 'regulatory_region_variant',
+        acc:'SO:0001566',
+        color: '#A52A2A',
+        impact: 'MODIFIER',
+        description:'A sequence variant located within a regulatory region'
+    },
+    'feature_elongation': {
+        name: 'feature_elongation',
+        acc:'SO:0001907',
+        color: '#7F7F7F',
+        impact: 'MODIFIER',
+        description:'A sequence variant that causes the extension of a genomic feature, with regard to the reference sequence'
+    },
+    'feature_truncation': {
+        name: 'feature_truncation',
+        acc:'SO:0001906',
+        color: '#7F7F7F',
+        impact: 'MODIFIER',
+        description:'A sequence variant that causes the reduction of a genomic feature, with regard to the reference sequence'
+    },
+    'intergenic_variant': {
+        name: 'intergenic_variant',
+        acc:'SO:0001628',
+        color: '#636363',
+        impact: 'MODIFIER',
+        description:'A sequence variant located in the intergenic region, between genes'
+    }
+};
+
+consequenceTypeDetails = consequenceTypeTreeFormat(consequenceTypeDetails);
+function consequenceTypeTreeFormat(data){
```

Comment:
Then there should be 2 files, one for the model and one for centralized view configuration.